app/views/layouts/application.html.haml

```diff
@@ -4,11 +4,13 @@
     %meta{ content: 'text/html; charset=UTF-8', 'http-equiv': 'Content-Type' }/
     %title
       = yield(:page_title) || 'Report management information to CCS'
-
     %meta{ name: 'viewport', content: 'width=device-width, initial-scale=1' }/
     %meta{ 'http-equiv': 'x-ua-compatible', content: 'ie-edge' }/
     %meta{ name: 'format-detection', content: 'telephone=no' }/
 
+    = favicon_link_tag('favicon.ico')
```

Comment:
These govuk-frontend assets get compiled with their folder-structure intact, so this line will need to reference them with the full nesting, for it to work in production where live asset compilation is turned off, e.g.

```suggestion
    = favicon_link_tag('govuk-frontend/assets/images/favicon.ico')
```

Note: you can test this in development mode by disabling the asset-pipeline with `config.assets.compile = false` in development.rb: